Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -194,11 +219,16 @@ namespace EMotionFX
         if (parent.isValid())
         {
             const Node* parentNode = static_cast<const Node*>(parent.internalPointer());
+
+            if (parentNode == m_characterRootNode) {
+                return static_cast<int>(m_skeleton->GetNumRootNodes());
```

Comment:
update: Brace position

Reply:
nit: Also we have a cast operator `aznumeric_cast` you can use instead of `static_cast` for these types of numerical conversions (helps catch lossy conversions)